packages/playground/src/components/manage_gateway_dialog.vue

```diff
@@ -0,0 +1,280 @@
+<template>
+  <div>
+    <v-dialog
+      model-value
+      @update:model-value="$emit('close')"
+      scrollable
+      width="70%"
+      :persistent="deleting || layout?.status === 'deploy'"
+    >
+      <weblet-layout
+        ref="layout"
+        @back="
+          () => {
+            gatewayTab = 0;
+            loadGateways();
+          }
+        "
+      >
+        <template #title>Manage Gateways </template>
+
+        <v-tabs align-tabs="center" color="primary" class="mb-6" v-model="gatewayTab" :disabled="deleting">
+          <v-tab>Gateway List</v-tab>
+          <v-tab>Add new gateway</v-tab>
+        </v-tabs>
+
+        <div v-show="gatewayTab === 0" :class="{ 'pb-2': !loadingGateways, 'pb-6': loadingGateways }">
+          <div class="d-flex justify-end mb-4">
+            <v-btn color="primary" :loading="loadingGateways" :disabled="deleting" @click="loadGateways" variant="tonal"
+              >Reload</v-btn
+            >
+          </div>
+          <list-table
+            :headers="[
+              { title: 'Name', key: 'name' },
+              { title: 'Contract ID', key: 'contractId' },
+              { title: 'Domain', key: 'domain' },
+              { title: 'TLS Passthrough', key: 'tls_passthrough' },
+              { title: 'Backends', key: 'backends' },
```

Comment:
we only support one backend on the gateway deployment
```suggestion
              { title: 'Backend', key: 'backends' },
```